nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/RequestTracingService.java

```diff
@@ -563,26 +574,70 @@ public RequestTraceStoreInterface getRequestTraceStore() {
     }
 
     @Override
-    public void collect(MonitoringDataCollector rootCollector) {
-        MonitoringDataCollector tracingCollector = rootCollector.in("trace");
+    public void collect(MonitoringWatchCollector collector) {
+        if ("true".equals(configuration.getEnabled())) {
```

Comment:
Reason there is for Yoda-style? <(°.°)>

Reply:
It is nullsafe and despite thera are two groups of developers, I like this more :D

Reply:
yes, its `null` safe, in contrast to `someVar.equals("stringConstant")`